### ISSUE TITLE
Using external libraries

### ISSUE THREAD

**Issue by jkstrawn** [godotjs/GodotJS#24]
Hello, I'm interested in porting over a tauri pixijs project to godot for better performance. This plugin seems very promising, but I'm getting errors while trying to use two of the three external libraries in my project. `matter-js` works fine, while `miniplex` and `radash` are giving me errors when loading the project in the editor. I'm using the prebuilt version of the Godot Editor as referenced in the v8 project.

While trying to use radash:
```
---
  modules\GodotJS\bridge\jsb_builtins.cpp:231 - [jsb][Error] can not load module 'radash' (with parent 'res://.godot/GodotJS/snake/snake.js') (User)
  modules\GodotJS\bridge\jsb_environment.cpp:1007 - [jsb][Warning] something went wrong on loading 'res://.godot/GodotJS/snake/snake.js'
  Uncaught Error: unknown module: radash
  Error: unknown module: radash
      at res:\\snake\snake.ts:4:26
```

example code:
```
import * as _ from "radash";

const max = _.max([4, 9, 2]);
```

While trying to use miniplex:
```
  modules\GodotJS\bridge\jsb_builtins.cpp:231 - [jsb][Error] can not load module 'miniplex' (with parent 'res://.godot/GodotJS/snake/snake.js') (User)
  Can't get file as string from path 'C:\...\GodotJSExample-main\node_modules\miniplex\dist\miniplex.cjs.js.map'.
  modules\GodotJS\bridge\jsb_environment.cpp:1007 - [jsb][Warning] something went wrong on loading 'res://.godot/GodotJS/snake/snake.js'
  Uncaught ReferenceError: process is not defined
  ReferenceError: process is not defined
      at C:\...\GodotJSExample-main\node_modules\miniplex\dist\miniplex.cjs.js:3:1
      at res:\\snake\snake.ts:4:26
```

example code:
```
import { World } from "miniplex";

const world = new World<Entity>()
```

Any help or suggestions would be very appreciated!

**Comment by jkstrawn** [comment]
I've gotten `miniplex` to work by adding a simple polyfill file that gets imported before importing the `miniplex` library:

```
(globalThis.process as any) = { env: { NODE_ENV: "development" } };
```

Still working on `radash` library

**Comment by ialex32x** [comment]
These issues are usually caused by the lack of nodejs support (libraries implemented in nodejs). The most viable solution is directly having a `node.impl` as an alternative javascript environment. Then, all nodejs stuff could be run in Godot, but I don't have a plan for this in the near future (maybe, after the `web.impl` is finished).

I haven't any experience in using `radash`, it should work if there are no nodejs relevant involved. I'll investigate this issue this weekend.

**Comment by ialex32x** [comment]
`radash` use `.cjs` as file ext which hasn't be considered yet. I'll fix it right away.

**Comment by ialex32x** [comment]
`radash` is available now (the module loader issue is fixed in the commit `9b2ab9f`)

**Comment by jkstrawn** [comment]
thank you!
